docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -207,114 +207,12 @@ def downstream_asset():
-```
-
-#### Defining basic dependencies for graph-backed assets
-
-The `from_graph` attribute on the `AssetsDefinition` object infers upstream and downstream asset dependencies from the graph definition provided. In the most simple case when the graph returns a singular output, Dagster infers the name of the graph to be the outputted asset key.
-
-In the example below, Dagster creates an asset with key `middle_asset` from the `middle_asset` graph. Just like assets defined via <PyObject object="asset" decorator />, each argument to the decorated graph function is an upstream asset name. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
-
-```python file=/concepts/assets/graph_backed_asset.py startafter=start_basic_dependencies endbefore=end_basic_dependencies
-from dagster import AssetsDefinition, asset, graph
-
-
-@asset
-def upstream_asset():
-    return 1
-
-
-@graph
-def middle_asset(upstream_asset):
-    return add_one(upstream_asset)
-
-
-middle_asset = AssetsDefinition.from_graph(middle_asset)
-
-
-@asset
-def downstream_asset(middle_asset):
-    return middle_asset + 1
-```
-
-When your graph returns multiple outputs, Dagster infers each output name to be the outputted asset key. In the below example, `two_assets_graph` accepts `upstream_asset` and outputs two assets, `first_asset` and `second_asset`:
-
-```python file=/concepts/assets/graph_backed_asset.py startafter=start_basic_dependencies_2 endbefore=end_basic_dependencies_2
-from dagster import AssetsDefinition, GraphOut, graph
-
-
-@graph(out={"first_asset": GraphOut(), "second_asset": GraphOut()})
-def two_assets_graph(upstream_asset):
-    one, two = two_outputs(upstream_asset)
-    return {"first_asset": one, "second_asset": two}
-
-
-two_assets = AssetsDefinition.from_graph(two_assets_graph)
-```
-
-#### Defining explicit dependencies for graph-backed assets
-
-You can also define dependencies for graph-backed assets explicitly via the `asset_keys_by_input_name` and `asset_keys_by_output_name` arguments to `from_graph`:
-
-```python file=/concepts/assets/graph_backed_asset.py startafter=start_explicit_dependencies endbefore=end_explicit_dependencies
-from dagster import AssetsDefinition, GraphOut, graph
-
-
-@graph(out={"one": GraphOut(), "two": GraphOut()})
-def return_one_and_two(zero):
-    one, two = two_outputs(zero)
-    return {"one": one, "two": two}
-
-
-explicit_deps_asset = AssetsDefinition.from_graph(
-    return_one_and_two,
-    keys_by_input_name={"zero": AssetKey("upstream_asset")},
-    keys_by_output_name={
-        "one": AssetKey("asset_one"),
-        "two": AssetKey("asset_two"),
-    },
-)
-```
+- [Graph-Backed Assets](/concepts/assets/graph-backed-assets): An asset whose computations are separated into multiple [ops](/concepts/ops-jobs-graphs/ops) that are combined to build a [graph](/concepts/ops-jobs-graphs/graphs). A graph-backed asset can output multiple assets.
```

Comment:
I think there's some overlap in these concepts that's worth pointing out here.  I would think of a graph-backed asset that outputs multiple assets as a kind of multi-asset.

The current wording might lead people to wonder about the difference between "A graph-backed asset can output multiple assets." and "A set of software-defined assets that are all updated by the same".

Reply:
Changing it to this:

- [Multi-Assets](/concepts/assets/multi-assets): A set of software-defined assets that are all updated by the same [op](/concepts/ops-jobs-graphs/ops) or [graph](/concepts/ops-jobs-graphs/graphs).
- [Graph-Backed Assets](/concepts/assets/graph-backed-assets): A specific type of multi-asset whose computations are separated into multiple [ops](/concepts/ops-jobs-graphs/ops) that are combined to build a [graph](/concepts/ops-jobs-graphs/graphs).

Hopefully this is a little clearer

Reply:
This makes it sound like all graph-backed assets are multi-assets, but I think it's more like a venn diagram where some graph-backed assets are multi-assets, some graph-backed assets aren't multi-assets, and some multi-assets aren't graph-backed assets.

Reply:
Updated instead to this text:

- [Multi-assets](/concepts/assets/multi-assets): A set of software-defined assets that are all updated by the same [op](/concepts/ops-jobs-graphs/ops) or [graph](/concepts/ops-jobs-graphs/graphs).
- [Graph-backed assets](/concepts/assets/graph-backed-assets): An asset whose computations are separated into multiple [ops](/concepts/ops-jobs-graphs/ops) that are combined to build a [graph](/concepts/ops-jobs-graphs/graphs). If the graph outputs multiple assets, the graph-backed asset is a [multi-asset](/concepts/assets/multi-assets).

Reply:
This looks great.